SeleniumCookie/wrapper.py

```diff
@@ -0,0 +1,373 @@
+# -*- coding: utf-8 -*-
+
+import os
+import os.path
+import sys
+import time
+import glob
+import http.cookiejar
+import tempfile
+import lz4.block
+import datetime
+import configparser
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+try:
+
+    from pysqlite2 import dbapi2 as sqlite3
+except ImportError:
+    import sqlite3
+
+# external dependencies
+import keyring
+import pyaes
+from pbkdf2 import PBKDF2
+
```

Comment:
Group all the default packages like sys is etc together and all the third party packages together. This is a good practice!